documentation/status-logging.md

```diff
@@ -0,0 +1,54 @@
+# Status logging
+
+TK-status can periodically log status data to a file in JSON format.
+
+To take advantage of this you must do two things:
+
+1. Enable logging though you TK app's configuration. For example, to log every 2 seconds, your configuration might look like this:
+
+    ```
+    status: {
+        logging-frequency: 2000,
+        status-logging-enabled: true
+    }
```

Comment:
Most of our other services have config settings that are `metrics` or `webrouting` or whatever, without `-service`, so just `status` seems the most reasonable.

I'm more in favor of no `enabled` setting - it seems like that will just end up making configuring it more complicated. Are there ever any other settings that the logging stuff would ever have? Would we ever have something about what status endpoints are logged or something? If we do think that we would ever add any logging-related settings, then maybe have like:
```
status: {
    debug-logging: {
         interval-seconds: 60
    }
}
```

or something like that?

I think I like calling it `debug-logging` over `status-logging`. I think.